EnglishFilter/sections/specific.txt

```diff
@@ -294,6 +294,10 @@ peazip.github.io##td[style="background-color: rgb(255, 255, 255); vertical-align
 kongregate.com##div[id^="rec_spot_ad_"]
 pairom.com###system_box_rotation
 pairom.com##.dimmer
+gaytail.com##.happy-footer
+gaytail.com##.happy-under-player
+gaytail.com##.wps-player__happy-inside
```

Comment:
We have common rules.
```
##.happy-under-player
##.wps-player__happy-inside
```

Also add the domain to the scriptlet rule `#%#//scriptlet("prevent-addEventListener", "getexoloader")`